cmd/snowflake.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 The Terraformer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package cmd
+
+import (
+	snowflake_terraforming "github.com/GoogleCloudPlatform/terraformer/providers/snowflake"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+	"github.com/spf13/cobra"
+)
+
+func newCmdSnowflakeImporter(options ImportOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "snowflake",
+		Short: "Import current State to terraform configuration from Snowflake",
+		Long:  "Import current State to terraform configuration from Snowflake",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			provider := newSnowflakeProvider()
+			err := Import(provider, options, []string{})
```

Comment:
you can just 
```suggestion
			return Import(provider, options, []string{})
```
and remove the next `if` block